max_vel: 17.00
max_acc: 10.00

kp_pos_h: 2.8
kp_vel_h: 2.5
ki_vel_h: 0.0
kd_vel_h: 0.0

kp_pos_z: 2.0
kp_vel_z: 2.0
ki_vel_z: 0.0
kd_vel_z: 0.0

kp_pose: 5.0

set_param_flag: FALSE
debug_pos_x: 2.0
debug_pos_y: -2.0
debug_pos_z: -3.0

# route: [0.00, 14.87,  37.68,  67.39, 93.00, 106.50, 112.70, 120.80,  121.00,  120.00,
#         0.00, -1.50, -11.80, -11.70, -9.40, -20.00, -35.67, -67.73,  -96.00, -110.00,
#         0.00, -2.24,  -1.02,  -0.40,  0.12,  -0.15,  -0.44,  -4.00,   -7.29,   -7.39]

route: [0.00, 1.487,  3.768,  6.742, 9.444, 10.650, 11.395, 12.177,  12.180,  12.000,
        0.00, -0.79, -1.227, -1.294, -0.805, -2.000, -3.570, -6.773,  -9.600, -11.000,
        0.00, 2.74,  1.02,  0.30,  0.02,  0.15,  0.34,  3.83,   7.39,   7.39]

ts: [2.80, 1.95, 2.10, 1.70, 1.40, 1.20, 2.10, 1.70, 2.00]

# ts: [5.60, 4.00, 4.00, 3.40, 2.80, 2.40, 4.20, 3.40, 4.00]

delay: 1.25
